presto-main/src/test/java/com/facebook/presto/block/BlockAssertions.java

```diff
@@ -368,6 +371,27 @@ public static Block createArrayBigintBlock(Iterable<? extends Iterable<Long>> va
         return builder.build();
     }
 
+    public static Block createObjectsArrayBlock(Type type, Object... object)
```

Comment:
`object` is probably not a good variable name. Generally array variables should have plural names, plus `object` isn't very descriptive